OracleHTTPServer/dockerfiles/12.2.1.2.0/Dockerfile

```diff
@@ -4,84 +4,97 @@
 #
-# (2) server-jre-8uXX-linux-x64.tar.gz
-#     Download from http://www.oracle.com/technetwork/java/javase/downloads/server-jre8-downloads-2133154.html
-#
 # HOW TO BUILD THIS IMAGE
 # -----------------------
 # Put all downloaded files in the same directory as this Dockerfile
 # As root user run following command manually:
 #
-#     $ sh buildDockerImage.sh -v 12.2.1
+#     $ sh buildDockerImage.sh -v 12.2.1.2.0
 #
 # Alternatively you can also run
 # 
 #     $ docker build --force-rm=true --no-cache=true --rm=true -t oracle/$IMAGE_NAME:$VERSION-$INSTALL_TYPE .
 #
 # For example
 # 
-#     $ docker build --force-rm=true --no-cache=true --rm=true -t oracle/ohs:12.2.1-sa .
+#     $ docker build --force-rm=true --no-cache=true --rm=true -t oracle/ohs:12.2.1.2.0-sa .
 #
-# Pull Oracle Linux 7 base image
+# Pull serverjre:8 base image
 # -----------------------------------
 FROM oracle/serverjre:8
 
 # Maintainer
 # ------------------------------------
-MAINTAINER Hemastuti Baruah <hemastuti.baruah@oracle.com>
+MAINTAINER Prabhat Kishore <prabhat.kishore@oracle.com>
 
 # Environment variables required for this build (do NOT change)
 # ----------------------------------------------
-ENV OHS_PKG=fmw_12.2.1.0.0_ohs_linux64_Disk1_1of1.zip \
-    OHS_BIN=fmw_12.2.1.0.0_ohs_linux64.bin \
+ENV OHS_PKG=fmw_12.2.1.2.0_ohs_linux64_Disk1_1of1.zip \
+    OHS_BIN=fmw_12.2.1.2.0_ohs_linux64.bin \
     ORACLE_HOME=/u01/oracle/ohssa \
     USER_MEM_ARGS="-Djava.security.egd=file:/dev/./urandom" \ 
-    PATH=$PATH:/usr/java/default/bin:/u01/oracle/ohssa/oracle_common/common/bin
+    PATH=$PATH:/usr/java/default/bin:/u01/oracle/ohssa/oracle_common/common/bin \
+    NM_PORT=5556 \
+    OHS_LISTEN_PORT=7777 \
+    OHS_SSL_PORT=4443 \
+    JAVA_HOME=/usr/java/default \
+    MW_HOME=/u01/oracle/ohssa \
+    DOMAIN_NAME=ohsDomain \
+    OHS_COMPONENT_NAME=ohs_sa1 \
+    PATH=$PATH:$ORACLE_HOME/oracle_common/common/bin:$ORACLE_HOME/user_projects/domains/ohsDomain/bin:/u01/oracle/container-scripts
+
 
 # Copy packages to the new drive
 # -------------------------------------------------------
 COPY $OHS_PKG install.file oraInst.loc /u01/
 
-# Install OL7 required packages. Refer FMW 12.2.1 System requirements guide for complete list of packages
+# Install OL7 required packages. Refer FMW 12.2.1.2 System requirements guide for complete list of packages
 # Change the open file limits and kernel parameters that need changing
 # Setup filesystem and oracle user
 #---------------------------------------------------------------------------------------------------------
 # Proxy needs to be set before running yum in /etc/yum.conf
 RUN yum install -y \
-        unzip tar nfs-utils binutils compat-libcap1 compat-libstdc++ \
-        gcc gcc-c++ glibc glibc-devel libaio libaio-devel libgcc \
-	libstdc++ libstdc++-devel ksh make ocfs2-tools sysstat && \
+        unzip libaio sysstat && \
```

Comment:
Java cannot be used to unzip archives that contain shell scripts. Unfortunately, JAR does not restores unix file permissions, and that may ruin everything. Leave `unzip` in if you are extracting something that has executable files.